### ISSUE TITLE
Visually appearing heading 'Columbia' text is not defined as heading in windows.

### ISSUE THREAD

**Issue by msftedad** [telerik/kendo-react#3478]
### Description

**Test Environment:**
OS: Windows 11 Version 25H2 (OS Build 26220.7271)
Browser: Edge (Version 143.0.3650.46)
Screen Reader: NVDA (2025.3.1)
URL: https://www.telerik.com/kendo-angular-ui/components/layout/expansionpanel/title

**User Impact:**
Screen reader users will be impacted because the visually prominent 'Columbia' text is not marked as a heading. This makes navigation harder, as they cannot use heading shortcuts to locate this section efficiently.

### Plan

KendoReact

### Steps To Reproduce

**Pre-Requisites:**
Enable NVDA

**Repro Steps:**

1. Open the URL: https://www.telerik.com/kendo-angular-ui/components/layout/expansionpanel/title in edge browser in windows.
2. 'Title' page will be open.
3. Press 'H' key in heading mode to move the headings.
4. verify whether visually appearing heading 'Columbia' text is defined as heading in windows or not.

### Screenshots or video

**Refer Attachment:**

1. Visually appearing heading 'Columbia' text is not defined as heading in windows.png
2. Visually appearing heading 'Columbia' text is not defined as heading in windows.mp4

### Actual Behavior

Visually appearing heading 'Columbia' text is not defined as heading in windows.

### Expected Behavior

Visually appearing heading 'Columbia' text should be defined as heading in windows.

### Browser

Microsoft Edge

### Browser version

Edge (Version 143.0.3650.46)

### OS type

Windows

### OS version

Windows 11 Version 25H2 (OS Build 26220.7271)
